.github/workflows/apk.yml

```diff
@@ -1,16 +1,16 @@
-name: Android Build
+name: Android Release Build
 
 on:
   workflow_dispatch:
     inputs:
       name:
         description: "Ex: milestone name"
-        default: "Mn"
+        default: "TestRelease"
```

Comment:
You must give it a name that is not a “test”, as it will be merged with the main file.

Reply:
Thank you for catching this, I will change it.